tbx/blog/migrations/0031_blogindexpage_page_links.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.2.5 on 2025-10-22 09:34
```

Comment:
forgot to add earlier: could you squash these into one? probably easiest is to remoe and regenerate